google-cloud-spanner-hibernate-performance-testing/src/main/java/com/google/cloud/spanner/hibernate/ClientLibraryDriver.java

```diff
@@ -0,0 +1,27 @@
+package com.google.cloud.spanner.hibernate;
```

Comment:
Missing copyright header? Is the checkstyle working?

Reply:
Done. My bad; yeah checkstyle is working.